IdentityCore/src/util/NSString+MSIDExtensions.m

```diff
@@ -259,14 +259,21 @@ - (BOOL)msidIsEquivalentWithAnyAlias:(NSArray<NSString *> *)aliases
     {
         return NO;
     }
-
+    MSID_LOG_WITH_CTX(MSIDLogLevelInfo, nil, @"Check if credential environment %@ is included in the list of valid environment aliases.",self);
```

Comment:
self is authority I assume. It might have some accidental PII in it, so let's only log host part of it?

Reply:
Sounds good.